### PR TITLE
Add BlockBundle config instructions and removing obsolete MarkItUpBun…

### LINKED ISSUE #338: Unknown function sonata_block_render_event
#### Sonata packages

``` bash
$ composer show sonata-project/*
sonata-project/admin-bundle              3.7.1  The missing Symfony Admin Generator
sonata-project/block-bundle              3.1.1  Symfony SonataBlockBundle
sonata-project/cache                     1.0.7  Cache library
sonata-project/classification-bundle     3.1.0  Symfony SonataClassificationBundle
sonata-project/core-bundle               3.1.1  Symfony SonataCoreBundle
sonata-project/datagrid-bundle           2.2    Symfony SonataDatagridBundle
sonata-project/doctrine-extensions       1.0.2  Doctrine2 behavioral extensions
sonata-project/doctrine-orm-admin-bundle 3.1.0  Symfony Sonata / Integrate Doctrine ORM into the SonataAdminBundle
sonata-project/easy-extends-bundle       2.1.10 Symfony SonataEasyExtendsBundle
sonata-project/exporter                  1.7.0  Lightweight Exporter library
sonata-project/formatter-bundle          3.0.1  Symfony SonataFormatterBundle
sonata-project/google-authenticator      1.0.2  Library to integrate Google Authenticator into a PHP project
sonata-project/intl-bundle               2.2.4  Symfony SonataIntlBundle
sonata-project/media-bundle              3.3.0  Symfony SonataMediaBundle
sonata-project/news-bundle               3.0.0  Symfony SonataNewsBundle
sonata-project/notification-bundle       3.0.0  Symfony SonataNotificationBundle
sonata-project/user-bundle               3.0.1  Symfony SonataUserBundle

```
#### Symfony packages

``` bash
$ composer show symfony/*
symfony/assetic-bundle     v2.8.0  Integrates Assetic into Symfony2
symfony/monolog-bundle     2.11.1  Symfony MonologBundle
symfony/phpunit-bridge     v2.8.4  Symfony PHPUnit Bridge
symfony/polyfill-apcu      v1.1.1  Symfony polyfill backporting apcu_* functions to lower PHP versions
symfony/polyfill-intl-icu  v1.1.1  Symfony polyfill for intl's ICU-related data and classes
symfony/polyfill-mbstring  v1.1.1  Symfony polyfill for the Mbstring extension
symfony/polyfill-php54     v1.1.1  Symfony polyfill backporting some PHP 5.4+ features to lower PHP versions
symfony/polyfill-php55     v1.1.1  Symfony polyfill backporting some PHP 5.5+ features to lower PHP versions
symfony/polyfill-php56     v1.1.1  Symfony polyfill backporting some PHP 5.6+ features to lower PHP versions
symfony/polyfill-php70     v1.1.1  Symfony polyfill backporting some PHP 7.0+ features to lower PHP versions
symfony/polyfill-util      v1.1.1  Symfony utilities for portability of PHP codes
symfony/security-acl       v2.8.0  Symfony Security Component - ACL (Access Control List)
symfony/swiftmailer-bundle v2.3.11 Symfony SwiftmailerBundle
symfony/symfony            v2.8.4  The Symfony PHP framework
```
#### PHP version

``` bash
$ php -v
PHP 7.0.11 (cli) (built: Sep 16 2016 20:15:56) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies
    with Zend OPcache v7.0.11, Copyright (c) 1999-2016, by Zend Technologies
    with Xdebug v2.4.1, Copyright (c) 2002-2016, by Derick Rethans
```
## Subject

Unknown function sonata_block_render_event
## Steps to reproduce

1 - Install sonata news with dependencies as per installation manual
2 - open localhost/news 
## Expected results

News page to open 
## Actual results

Error is shown

```
Unknown "sonata_block_render_event" function in SonataNewsBundle:Post:archive.html.twig at line 18.
500 Internal Server Error - Twig_Error_Syntax
```

### PR DESCRIPTION
**Pull request by bizmate**
I am targeting this branch, because i have seen doc updates pointing to master

Fixes #338 
## Changelog

``` markdown

### Changed
- Installation instructions for the BlockBundle dependency
```
## To do

- [x] Update the documentation
## Subject

Add missing step to configure the BlockBundle and remove obsolete MarkItUp Bundle reference

### CODE REVIEW

File: Resources/doc/reference/installation.rst [hunk 1]
```diff
@@ -151,6 +151,28 @@ if you use the API.
...
+            #sonata.admin_doctrine_orm.block.audit:
+            #    contexts:   [admin]
+
+            sonata.block.service.text:
+            sonata.block.service.rss:
+
+            # Some specific block from the SonataMediaBundle
+            sonata.media.block.media:
+            sonata.media.block.gallery:
+            sonata.media.block.feature_media:
```
Comment (greg0ire) [inline Resources/doc/reference/installation.rst]: Why not link to the block docs instead?

File: Resources/doc/reference/installation.rst [hunk 2]
```diff
@@ -151,7 +151,7 @@ if you use the API.
                     - sonata.formatter.twig.gist
                     - sonata.media.formatter.twig
 
-* Create configuration file ``sonata_block.yml`` for block rendering:
+* Create configuration file ``sonata_block.yml`` for block rendering as per [Sonata Block Configuration]:
```
Comment (core23) [inline Resources/doc/reference/installation.rst]: This file must be included in ``config.php``.

Comment (bizmate) [inline Resources/doc/reference/installation.rst]: That obvious step is also skipped in the documentation after the first example. ie, 

The import example is given for the first config/yml only at https://github.com/bizmate/SonataNewsBundle/blame/d5856b005690283ac152492f777a20ce27492f35/Resources/doc/reference/installation.rst#L92

In the other configs is not included and it becomes repetitive. If added should be either left only once at the top or put everywhere a config is described. For instance there is not need to repeat that the import is required here
https://github.com/bizmate/SonataNewsBundle/blame/d5856b005690283ac152492f777a20ce27492f35/Resources/doc/reference/installation.rst#L105

or 

https://github.com/bizmate/SonataNewsBundle/blame/d5856b005690283ac152492f777a20ce27492f35/Resources/doc/reference/installation.rst#L122

Comment (core23) [inline Resources/doc/reference/installation.rst]: Then we should change this 👍

File: Resources/doc/reference/installation.rst [hunk 3]
```diff
@@ -81,28 +80,11 @@ if you use the API.
...
-            types:
-                json: Sonata\Doctrine\Types\JsonType
-
-
 * Add a new context into your ``sonata_media.yml`` configuration if you don't have go there https://sonata-project.org/bundles/media/master/doc/reference/installation.html:
 
 .. code-block:: yaml
 
-    # app/config/sonata_media.yml
+    # app/config/sonata_media.yml.
```
Comment (core23) [inline Resources/doc/reference/installation.rst]: Please remove the dot here

Comment (bizmate) [inline Resources/doc/reference/installation.rst]: removed earlier today, could you pls review?

**Comment by core23** [comment]
Please rebase @bizmate

**Comment by OskarStark** [comment]
ping @bizmate

**Comment by greg0ire** [comment]
Please rebase and when you do, use `git push --force`

**Comment by bizmate** [comment]
@greg0ire this is from a rebase, i also changed https://github.com/sonata-project/SonataNewsBundle/pull/339/commits/eae14cd4801cb434141e7084879eae6916635540#diff-76c65fb57d60a50b59c152ff7b22b427R238 and https://github.com/sonata-project/SonataNewsBundle/pull/339/commits/eae14cd4801cb434141e7084879eae6916635540#diff-76c65fb57d60a50b59c152ff7b22b427R238 hence the new commit with conflict resolved

**Comment by core23** [comment]
Can you please rebase @bizmate 
Also please check if this is BC, so you can merge against the latest 3.x stable branch.

**Comment by core23** [comment]
Ping @bizmate

**Comment by OskarStark** [comment]
Closing in favour of #379, as we got no feedback from @bizmate

**Pull request closed without merging**